apps/website/src/components/courses/CourseSearchCard.tsx

```diff
@@ -29,25 +36,40 @@ export const CourseSearchCard: React.FC<CourseSearchCardProps> = ({
     <a
       href={url}
       className={clsx(
-        'course-search-card flex flex-col md:flex-row gap-6 container-lined p-6 max-w-[828px] size-full md:max-h-[320px] transition-transform duration-200 hover:scale-[1.01] hover:container-elevated',
+        'course-search-card flex sm:flex-row flex-col gap-6 container-lined p-6 max-w-[828px] size-full transition-transform duration-200 hover:scale-[1.01] hover:container-elevated',
         className,
       )}
     >
-      <div className="course-search-card__image-container h-[240px] w-full md:w-fit">
+      <div className="course-search-card__image-container h-[240px] sm:w-fit w-full">
         <img className="course-search-card__image size-full object-cover rounded-lg" src={imageSrc || defaultImageSrc} alt={title} />
       </div>
-      <div className="course-search-card__content flex flex-col gap-4 flex-1">
+      <div className="course-search-card__content flex flex-col justify-between gap-space-between flex-1">
         <div className="course-search-card__text">
-          <H3 className="course-search-card__title mb-6">
+          <H3 className="course-search-card__title mb-4">
             {title}
           </H3>
           {description && (
-            <P className="course-search-card__description overflow-hidden text-bluedot-black text-ellipsis mb-6">
+            <P className="course-search-card__description text-size-s overflow-hidden text-bluedot-black text-ellipsis mb-6">
```

Comment:
```suggestion
            <P className="course-search-card__description overflow-hidden text-bluedot-black text-ellipsis mb-6">
```

`text-size-s` isn't a valid class (and never was, although we used it a bunch)

The valid thing is `text-size-sm`, which P automatically comes with already

Reply:
Ah good catch, thanks for fixing 🙏